app/controllers/squad_controller.rb

```diff
@@ -40,17 +40,20 @@ def update
     user = User.find_by("email = '#{params[:email]}'")
     @squad = Squad.find(params[:id]) #TODO : something along those lines
     authorize @squad
-    Squadmember.new(user: user, squad: @squad, squadchosenvenue_id: @squad.squadchosenvenues.first.id).save! #TODO : something along those lines
+    @squadmember = Squadmember.new(user: user, squad: @squad, squadchosenvenue: @squad.squadchosenvenues.first, contribution: 0) #TODO : something along those lines
+    @squadmember.save!
+    respond_to do |format|
+    format.js
+    end
   end
   #AJAX REQUESTS
 
   def confirm_squad_member
     @squad = Squad.find(params[:id])
-
    squadmember = Squadmember.find_by('user_id current_user') # TODO : figure activerecord call to find squadmember id in the good squad and the good user_id
-
+   squadmember.squadchosenvenue_id = 0 (params[:venue_id])
```

Comment:
incomplete, needed to add logic to make it overwrite previous choice if user decides to vote on something else